nvtabular/columns/schema.py

```diff
@@ -36,7 +137,32 @@ def with_tags(self, tags) -> "ColumnSchema":
 
         tags = list(set(list(self.tags) + tags))
 
-        return ColumnSchema(self.name, tags=tags)
+        return ColumnSchema(self.name, tags=tags, properties=self.properties)
```

Comment:
I think this loses some information (dtype/is_list). how about: 

```suggestion
        return ColumnSchema(self.name, tags=tags, properties=self.properties, dtype=self.dtype, _is_list=self._is_list)
```